src/assessments/seo/textImagesAssessment.js

```diff
@@ -80,19 +125,20 @@ class TextImagesAssessment extends Assessment {
 			};
 		}
 
-		// Has alt-tag and keywords
-		if ( this.altProperties.withAltKeyword > 0 ) {
+		// Has alt-tags, but no keyword is set.
+		if ( this.altProperties.withAlt > 0 ) {
 			return {
-				score: this._config.scores.withAltKeyword,
+				score: this._config.scores.withAlt,
 				resultText: i18n.dgettext(
 					"js-text-analysis",
-					"The images on this page contain alt attributes with the focus keyword."
+					"The images on this page contain alt attributes. " +
+					"Once you've set a focus keyword, also include the keyword where appropriate."
```

Comment:
The feedback string is not clear about where to include the keyword. Maybe "Once you've set a focus keyword, don't forget to include it in alt attributes, where appropriate".

Reply:
Fixed